external-report-demo/report-demo/app.js

```diff
@@ -17,33 +17,43 @@ let response;
  */
 exports.lambdaHandler = async (event, context) => {
     try {
-      const json = validateJSON(event);
+      const body = validateJSON(event);
 
-      const renderCSV = (key) => {
-        if (json[key] && json[key].length > 0) {
-          message = Object.keys(json[key][0]).join(",") + "\n";
-          message += json[key].map(item => Object.values(item).join(",")).join("\n");
-        } else {
-          message = `${key.charAt(0).toUpperCase() + key.slice(1)} report requested, but no ${key} were in the query.\n\n`
-          message += "Full query:\n\n"
-          message += json;
+      if (body.jwt) {
+        body.json = JSON.parse(body.json);
+        body.parsedJWT = JSON.parse(Buffer.from(body.jwt.split('.')[1], 'base64').toString());
```

Comment:
It would be best if this demo report was extended to actually request the learners using the JWT. This way we can test this new API without needing to use the query creator. 

If you want to put that in a different PR, that would be fine. I put a task in the story so we don't forget this.

Reply:
Sure. I had been thinking maybe it didn't make sense, because we don't want this demo app to be uploading anything to S3, so we don't want it to load more learners than can fit in memory, which means we may not be able to test out pagination.

But just testing even a single small batch of getting learners via JWT would be enough to show part of how to use the token.

Also, if we make the change that the external report is the one setting the pagination count, then this could use a smaller value.

Reply:
The report is updated to show the multi-page round-trip. It uses the new query parameter of setting page_size, so we can demo pagination with small numbers of learners.

<img width="1023" alt="Screen Shot 2021-05-27 at 5 48 56 PM" src="https://user-images.githubusercontent.com/35721/119901435-e871fb80-bf13-11eb-8e92-35ce41adf42c.png">